libs/platform/src/lib/list/list.component.ts

```diff
@@ -430,6 +434,14 @@ export class ListComponent<T> extends CollectionBaseInput implements OnInit, Aft
         indicator?.setAttribute('aria-label', '');
     }
 
+    /** @hidden */
+    ngAfterViewChecked(): void {
+        if (this.hasObject) {
+            // object lists should never be compact
+            this._ulElement?.classList.remove('fd-list--compact');
```

Comment:
i think it should be done in styles like so

```css
.fd-list--compact:not(.fd-object-list) { ... }
```

Reply:
That is a better idea! Let me give it a try, thanks